README.md

```diff
@@ -1,37 +1,34 @@
 # wkhtmltox-sys
 
-Auto-generated bindings to libwkhtmltox. Depends on libwkhtmltox being installed.
-
-Resource  | Link    
------ | -----
-Crate | [![Crates.io](https://img.shields.io/crates/v/wkhtmltox-sys.svg?maxAge=2592000)](https://crates.io/crates/wkhtmltox-sys)
-Documentation | [Cargo docs](https://anowell.github.io/wkhtmltox-sys/wkhtmltox_sys/)
-High-level bindings | [wkhtmltopdf-rs](https://github.com/anowell/wkhtmltopdf-rs)
-Upstream | [wkhtmltopdf.org](http://wkhtmltopdf.org/)
+Auto-generated bindings to libwkhtmltox.
 
+| Resource            | Link                                                                                                                     |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| Crate               | [![Crates.io](https://img.shields.io/crates/v/wkhtmltox-sys.svg?maxAge=2592000)](https://crates.io/crates/wkhtmltox-sys) |
+| Documentation       | [Cargo docs](https://anowell.github.io/wkhtmltox-sys/wkhtmltox_sys/)                                                     |
+| High-level bindings | [wkhtmltopdf-rs](https://github.com/anowell/wkhtmltopdf-rs)                                                              |
+| Upstream            | [wkhtmltopdf.org](http://wkhtmltopdf.org/)                                                                               |
 
 ## Generating
 
 This is generated with bindgen:
 
-```
-$ bindgen --link=dynamic=wkhtmltox include/pdf.h > src/pdf.rs
-$ bindgen --link=dynamic=wkhtmltox include/image.h > src/image.rs
-$ cargo fmt
+```sh
+bindgen --dynamic-loading wkhtmltox include/pdf.h > src/pdf.rs -- -DBUILDING_WKHTMLTOX=1
+bindgen --dynamic-loading wkhtmltox include/image.h > src/image.rs -- -DBUILDING_WKHTMLTOX=1
+cargo fmt
 ```
 
 ## Examples
 
 The Rust example is built with cargo:
 
-```
-cargo test
-target/debug/examples/convert-string
+```sh
+cargo build --example convert_string
+./target/debug/examples/convert_string
 ```
 
-The C example can be built with gcc:
-
-```
-gcc -g -Wall examples/convert-string.c -o target/convert-string-c -lwkhtmltox
-target/convert-string-c
+```sh
+cargo build --example convert_url
```

Comment:
is there a particular reason for renaming the files to use underscores? I subscribe to the idea of using underscores in identifiers where ambiguity may cause them to be interpreted as a minus, but otherwise, that hyphens are nicer for naming - hence using hyphens in filenames.

Reply:
It is more common to see Rust programmers using underscores, but no problem about it too, the more important thing is the dynamic loading possibility.